packages/flutter/test/material/text_field_test.dart

```diff
@@ -3597,6 +3597,41 @@ void main() {
     semantics.dispose();
   });
 
+  testWidgets('Disabled text field hides helper and counter', (WidgetTester tester) async {
+    const String helperText = 'helper text';
+    const String counterText = 'counter text';
+    Widget buildFrame(bool enabled) {
+      return MaterialApp(
+        home: Material(
+          child: Center(
+            child: TextField(
+              decoration: InputDecoration(
+                labelText: 'label text',
+                helperText: helperText,
+                counterText: counterText,
+                enabled: enabled,
+              ),
+            ),
+          ),
+        ),
+      );
+    }
+
+    // When enabled is true, the helper and counter are visible.
```

Comment:
Shouldn't we be checking the errorText as well? https://api.flutter.dev/flutter/material/InputDecoration/enabled.html

Reply:
I tried to be lazy and just check one because they're the same widget under the hood.  I'll add a check for both though :)

Reply:
Done.